### PR TITLE
fix/VARNS-96: Initial configuration is added even when we load from a file

### PR DESCRIPTION
**Pull request by atran50**
https://tree.taiga.io/project/ser515asu-ser515-VARNS/issue/96

Fixes the default user stories being added each startup. Adds a button to US List for adding the default ones.

### CODE REVIEW

**Comment by atran50** [comment]
image

**Review by srchowd3-445 — APPROVED** [review]
Besides the accumulating Sprints for every new SImulation instantiated, everything else looks good!

File: src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/panels/DemoPane.java
Comment (srchowd3-445) [inline src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/panels/DemoPane.java:1]: image


Seems there's a merge conflict here

**Review by srchowd3-445 — APPROVED** [review]
LGTM boss

**Pull request merged**